pkg/util/mount/mount_windows_test.go

```diff
@@ -601,3 +601,93 @@ func TestGetFileType(t *testing.T) {
 		}
 	}
 }
+
+func TestIsLikelyNotMountPoint(t *testing.T) {
+	mounter := Mounter{"fake/path"}
+	base, err := ioutil.TempDir("", "TestIsLikelyNotMountPoint")
```

Comment:
Should you make the TempDir per test case so that you won't run into name collision issues?

Reply:
@msau42 I have already merged a PR: https://github.com/kubernetes/kubernetes/pull/63399 which fixes this issue, and also rebased on the latest master branch, now every test case in `mount_windows_test.go` will make TempDir

Reply:
I actually mean each test case in tests uses a new tmpdir.  That way you don't need to rely on each test case filename to be unique

Reply:
Hi @msau42 actually `ioutil.TempDir("", "")` will create a new tmpdir with a unique name, add a prefix like `"TestIsLikelyNotMountPoint"` is for debugging purpose, so when it fails, I could comment out `defer os.RemoveAll(base)` and find the tmp directory easily. Anyway, whether I set prefix or not, `ioutil.TempDir` will still create a unique dir.

Reply:
Sorry I think I'm not being clear. I was suggesting a new tempdir for each iteration in the testcase loop

Reply:
@msau42 good point, I have changed the test case, create a new tempdir for every single test case, PTAL